index.js

```diff
@@ -86,6 +85,8 @@ export default class ClapprStats extends ContainerPlugin {
   }
 
   onFirstPlaying() {
+    this.listenTo(this.container, Events.CONTAINER_TIMEUPDATE, (e) => this.container.playback.isPlaying() && this.onContainerUpdateWhilePlaying())
```

Comment:
This is a fine work around but I was talking with @towerz and we thought that we need to solve this problem on clappr's side https://github.com/clappr/clappr/blob/142e58f7d407635386d2ecc53df98b3249ef8866/src/playbacks/hls/hls.js#L173-L175 :) 

but I'll accept this one